src/lib/seam/devices/index.ts

```diff
@@ -1 +1,9 @@
+import type { LockDevice } from 'seamapi'
+
 export * from './use-devices.js'
+
+export const getLockModel = ({ properties }: LockDevice) => {
+  return (
+    properties?.august_metadata?.model ?? properties?.schlage_metadata?.model
```

Comment:
@seveibar Interesting that a consumer of the Seam API would need to implement this exhaustive fallback pattern over all device providers as well as come back and add new ones here. Is there a better interface for this, could there be?

Reply:
glad you mentioned this. the current `*_metadata.model` is a bad attempt at a standardization. Sy has been working to standardize the device db so that we have a representative model. I'm currently of the opinion that model should be a standard object not a standard string e.g. `device.properties.model = { color: "black", year: 2022, series_name: "YRD226" }` or something. But there is a lot of work to be done- it's not an easy problem unfortunately if you want certain assumptions around the model name to hold true (e.g. that two models are the same for purchasing where colors and trim matter vs the same for functionality)

Reply:
i think having logic like this is a great interim solution until we have a proper standardized model (which is a great RFC project)

Reply:
Good to know  this is on the radar!

For now, I'm wondering if we could make this function dynamic, e.g., loop through properties, and pick the first non-null `properties?.*_metadata.model`.

Reply:
that seems ok- although i'm almost wondering if we could do something like `device.properties.model = { display_name }` where you're not committing to functionality/trim/purchasing.

Reply:
Would this be adding to the hook so it returns a slightly extended type? Or on the API response?

Reply:
sorry on the api response- i think we could propose the model object and push that through relatively quickly.

Reply:
https://github.com/seamapi/react/issues/127